content/en/docs/Tasks/packaging-an-operator.md

```diff
@@ -233,6 +233,41 @@ TODO: example status
 
 `nativeAPIs` is an optional field, but the more information you give OLM about the context in which your operator should be run, the more informed decisions OLM can make.
 
+## Packaging Additional Objects Alongside an Operator
+
+Operators can include additional objects alongisde their `CSV` in the `/manifests` directory. These objects should be YAML files and valid kuberenetes objects. The following objects are supported as of OLM 0.16.0:
+* ConfigMaps
+* Secrets
+* Services
+* [PodDisruptionBudgets](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/adding-pod-disruption-budgets.md)
+* [PriorityClasses](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/adding-priority-classes.md)
+* [VerticalPodAutoscalers](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/adding-vertical-pod-autoscaler.md)
+
+*Note: some of these objects can affect upgrade of the cluster and potentially cause problems for workloads unrelated to your operator. Be sure to understand
+the safe use of these objects before packaging them with your operator. See the docs linked above for more information on these objects as they relate to OLM.* 
+
+### Limitations on Pod Disruption Budgets
+
+No limitations are placed on the contents of a PDB at this time when installing on-cluster, but that may change as OLM develops an advanced strategy to ensure installed objects do not compromise the cluster.
+
+However, the following are suggested guidelines to follow when including PDB objects in a bundle.
+
+* maxUnavailable field cannot be set to 0 or 0%.
+    * This can make a node impossible to drain and block important lifecycle actions like operator upgrades or even cluster upgrades.
+* minAvailable field cannot be set to 100%.
+    * This can make a node impossible to drain and block important lifecycle actions like operator upgrades or even cluster upgrades.
+
+### Limitations on Priority Classes
+
+No limitations are placed on the contents of a PriorityClass manifest at this time when installing on-cluster, but that may change as OLM develops an advanced strategy to ensure installed objects do not compromise the cluster.
```

Comment:
Keeping versioned documents in mind: do we want to stick to present tense here, and when we release future version of the doc intimate the users of limitations there? 

i.e 

`No limitations are placed on the contents of a PriorityClass manifest when installing on-cluster`

And in 0.17.0 version of the doc for example: 
`We have advanced strategy to ensure installed objects do not compromise the cluster, and that includes limitations on these objects`

Reply:
I think that's a good idea since we will have versioned docs. I will simply remove the second part of the sentence as you suggested.